### PR TITLE
Updated the two main custom Exceptions to not extend Throwable, but rather RuntimeException.

### PR DESCRIPTION
**Pull request by wdschei**

### CODE REVIEW

**Comment by Mario-Lopez** [comment]
Need to update RELEASE.md (which is missing its `## In Progress Work ##` section).

**Comment by RackerWilliams** [comment]
I don't see the value in this being 3 commits.

File: RELEASE.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
 # Releases #
+## Release IN PROGRESS (2016-11-??) ##
+1. Updated the two custom exceptions to no longer extend the base ```Throwable``` and instead the more restrictive ```RuntimeException```.
```
Comment (RackerWilliams) [inline RELEASE.md]: Should be something like:

1. Clean up : Convert '''Throwable''' to more restrictive ```RuntimeException```

Comment (wdschei) [inline RELEASE.md]: Updated.

**Comment by wdschei** [comment]
Commits squashed.

**Pull request merged**